### ISSUE TITLE
Adding icons to buttons causes performance issues

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#720]
Using the [icon] input to add an icon to Kendo buttons causes performance issues due to the setTimeout() used internally in the following function:

```ts
private prependChild(node: HTMLElement): void {
    setTimeout(() => {
      if (this.renderer && node !== this.element.firstChild) {
        this.renderer.insertBefore(this.element, node, this.element.firstChild);
      }
    });
  }
```

Internal information: Reported in Ticket ID: 1117699

**Comment by Neaco** [comment]
Hi, is it still ongoing? It makes it impractical to use the grid buttons.

**Comment by ipeshev** [comment]
Shipped with v4.3.0 of @progress/kendo-angular-buttons
